students/vmedina/lesson_08/inventory.py

```diff
@@ -0,0 +1,30 @@
+"""
+Victor Medina
+Assignment 8
+
+"""
+
+import os
+import csv
+from functools import partial
+
+
+def add_furniture(invoice_file, customer_name, item_code, item_description, item_monthly_price):
+    invoice_dir_path = os.path.dirname(os.path.realpath(__file__)) + invoice_file
```

Comment:
You're missing the separator between the path and the filename, which will cause the output filename to be _lesson_08rented_items.csv_. Try to also to include any support files you used. In my case, I tested your code with the following lines:

```
from inventory import *
add_furniture("rented_items.csv", "Elisa Miles", "LR04", "Leather Sofa", 25)
add_furniture("rented_items.csv", "Edward Data", "KT78", "Kitchen Table", 10)
add_furniture("rented_items.csv", "Alex Gonzales", "QM01", "Queen Mattress", 17)
create_invoice = single_customer("Susan Wong", "rented_items.csv")
create_invoice("test_items.csv")
```

Pylint is flagging your code for missing docstrings in a couple of places, should be an easy fix.

-Luis